modules/api/src/main/java/io/fluo/api/data/RowColumnValue.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2014 Fluo authors (see AUTHORS)
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.fluo.api.data;
+
+public class RowColumnValue extends RowColumn {
```

Comment:
Will this class be comparable as it extends RowColumn or does it need to implement `Comparable<RowColumnValue>`?